packages/formation-react/src/components/MegaMenu/SubMenu.jsx

```diff
@@ -1,16 +1,16 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import Column from './Column';
-import _ from 'lodash';
-import { ArrowRightBlueSVG } from './arrow-right-blue';
+import React from "react";
```

Comment:
It looks like the wrong Prettier rules got applied in here. @1Copenut You might need to get the latest and run the lint commands again (cc @rianfowler )

Reply:
[This](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/35#issue-263072113) should fix it once it's merged. Feel free to merge on approval.